package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "apostrophe",
-  "version": "3.64.0",
+  "version": "3.66.0",
```

Comment:
3.65.0 was a hotfix release, the code from it did reach main if you're wondering, just not the version number.